lib/tapioca/cli.rb

```diff
@@ -4,104 +4,113 @@
 module Tapioca
   class Cli < Thor
     include CliHelper
+    include ConfigHelper
 
-    class_option :outdir,
-      aliases: ["--out", "-o"],
-      banner: "directory",
-      desc: "The output directory for generated RBI files"
-    class_option :generate_command,
-      aliases: ["--cmd", "-c"],
-      banner: "command",
-      desc: "The command to run to regenerate RBI files"
-    class_option :file_header,
-      type: :boolean,
-      default: true,
-      desc: "Add a \"This file is generated\" header on top of each generated RBI file"
+    class_option :config,
+      aliases: ["-c"],
+      banner: "<config file path>",
+      type: :string,
+      desc: "Path to the Tapioca configuration file",
+      default: TAPIOCA_CONFIG
     class_option :verbose,
       aliases: ["-V"],
       type: :boolean,
-      default: false,
-      desc: "Verbose output for debugging purposes"
-
-    map T.unsafe(["--version", "-v"] => :__print_version)
+      desc: "Verbose output for debugging purposes",
+      default: false
 
     desc "init", "initializes folder structure"
     def init
       generator = Generators::Init.new(
-        sorbet_config: Config::SORBET_CONFIG,
-        default_postrequire: Config::DEFAULT_POSTREQUIRE,
-        default_command: Config::DEFAULT_COMMAND
+        sorbet_config: SORBET_CONFIG,
+        default_postrequire: DEFAULT_POSTREQUIRE,
+        default_command: DEFAULT_COMMAND
       )
       generator.generate
     end
 
     desc "require", "generate the list of files to be required by tapioca"
+    option :postrequire, type: :string, default: DEFAULT_POSTREQUIRE
```

Comment:
Depending on the answer to the previous questions:

```suggestion
    option :postrequire_path, type: :string, default: DEFAULT_POSTREQUIRE_PATH
```

Reply:
I chose to not change this, since the option name is long and I feel like we don't have much to gain by adding `_file` or `_path` at the end.